my_test_requests_delete.py

```diff
@@ -0,0 +1,11 @@
+import requests
```

Comment:
By the way, for testing our API, we can use `unittests` or `pytest` and check it programmatically.